sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeEntitiesTests.cs

```diff
@@ -228,5 +192,62 @@ public async Task RecognizeEntitiesBatchWithNullTextTest()
             InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[0].Entities.Count());
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
+
+        private void ValidateInDocumenResult(CategorizedEntityCollection entities, int minCount = 1)
+        {
+            Assert.IsNotNull(entities.Warnings);
+            Assert.GreaterOrEqual(entities.Count, minCount);
+            foreach (CategorizedEntity entity in entities)
+            {
+                Assert.That(entity.Text, Is.Not.Null.And.Not.Empty);
+                Assert.That(entity.Category, Is.Not.Null);
```

Comment:
Question: I see that we use both this form as well as the `Assert.IsNotNull` methods. Should we standardize on one?

Reply:
ahhh you are right! I had it as `Is.Not.Null.And.Not.Empty`, realized it could be `Empty` so deleted that part.
I will update to `Assert.IsNotNull()`